test/modellib/ModelLib.cpp

```diff
@@ -20,11 +20,12 @@
 #include "src/Runtime/OMTensorHelper.h"
 #include "test/modellib/ModelLib.hpp"
 
-using namespace std;
 using namespace mlir;
-using namespace onnx_mlir;
 
-ModelLibBuilder::ModelLibBuilder(const string &name)
+namespace onnx_mlir {
+namespace test {
```

Comment:
Curiosity: why do we use `using namespace mlir` but `namespace onnx_mlir { }`

Reply:
I prefer 'wrapping' declarations with `namespace { ... }` rather than adding a `using declaration` because, if a function is declared in the file, and is enclosed in `namespace { ... }` it is declared **in** the namespace. Conversely, is there was just a using declaration, the function would be declared in the global namespace, which is something we should avoid polluting.